### PR TITLE
docs: Add more Rust examples to User Guide

### PR DESCRIPTION
**Pull request by lukemanley**

### CODE REVIEW

File: docs/source/user-guide/expressions/strings.md [hunk 1]
```diff
@@ -118,9 +118,11 @@ the string:
...
 | `strip_suffix`      | Removes an exact substring suffix if present.                         |
 
-??? info "Similarity to Python string methods" `strip_chars` is similar to Python's string method
-`strip` and `strip_prefix`/`strip_suffix` are similar to Python's string methods `removeprefix` and
-`strip_suffix`, respectively.
+??? info "Similarity to Python string methods"
+
+    `strip_chars` is similar to Python's string method
+    `strip` and `strip_prefix`/`strip_suffix` are similar to Python's string methods `removeprefix` and
+    `strip_suffix`, respectively.
```
Comment (rodrigogiraoserrao) [inline docs/source/user-guide/expressions/strings.md:125]: Drive-by request:

```suggestion
    `removesuffix`, respectively.
```

Comment (lukemanley) [reply docs/source/user-guide/expressions/strings.md]: updated, thanks
